website/static/templates.json

```
@@ -5252,5 +5252,30 @@
       "python"
     ],
     "id": "72a85172-4c98-4f01-9127-b17c7f8afc88"
+  },
+  {
+    "title": "Azure Functions TypeScript CosmosDB Trigger",
+    "description": "An Azure Functions quickstart project that demonstrates how to use a Cosmos DB Trigger with Azure Developer CLI (azd) for quick and easy deployment, using TypeScript and Node.js. The sample uses managed identity and a virtual network to make sure deployment is secure by default.",
+    "preview": "./templates/images/functions-quickstart-typescript-azd-cosmosdb.png",
+    "authorUrl": "https://github.com/Azure/Azure-Samples",
```

Should this be:
```suggestion
    "authorUrl": "https://github.com/Azure-Samples",
```

https://github.com/Azure/Azure-Samples appears to be invalid and isn't used in any previous templates either